nevergrad/optimization/optimizerlib.py

```diff
@@ -1476,6 +1476,10 @@ def _learn_on_k_best(archive: utils.Archive[utils.MultiValue], k: int) -> tp.Arr
     X2 = polynomial_features.fit_transform(X)
 
     # Fit a linear model.
+    if not max(y) - min(y) > 1e-20:
+        raise MetaModelFailure
+
+    y = y / (max(y) - min(y))
```

Comment:
probably equivalent ? Not sure if it's worth it
```suggestion
    y = (y - min(y)) / (max(y) - min(y))
```